packages/esm-service-queues-app/translations/en.json

```diff
@@ -209,7 +210,7 @@
   "today": "Today",
   "totalPatients": "Total Patients",
   "transition": "Transition",
-  "TransitionLatestQueueEntry": "Transition patient to latest queue",
+  "transitionLatestQueueEntry": "Transition patient's latest queue entry",
```

Comment:
@denniskigen since the key is changed, shouldn't the `extract-translations` script have updated all the other language jsons?